packages/client/test/test-utils/utils.ts

```diff
@@ -163,8 +168,31 @@ export const createStreamRegistryCached = (opts: {
     } as any
 }
 
+export const createGroupKeyManager = (groupKeyStore: GroupKeyStore = mock<GroupKeyStore>()): GroupKeyManager => {
+    return new GroupKeyManager(
+        groupKeyStore,
+        mock<LitProtocolFacade>(),
+        mock<SubscriberKeyExchange>(),
+        new StreamrClientEventEmitter(),
+        new DestroySignal(),
+        {
+            encryption: {
+                litProtocolEnabled: false,
+                litProtocolLogging: false
+            },
+            decryption: {
+                maxKeyRequestsPerSecond: 10,
+                keyRequestTimeout: 0
```

Comment:
To keep the existing functionality of `Decrypt.ts`:

```suggestion
                keyRequestTimeout: 50
```

The value is not relevant for other test cases which currently use this fake `GroupKeyManager`.